sonar-orchestrator/src/main/java/com/sonar/orchestrator/OrchestratorBuilder.java

```diff
@@ -306,9 +306,11 @@ public OrchestratorBuilder activateLicense(String pluginKey) {
    * in 5.5, see https://jira.sonarsource.com/browse/SONAR-7122 and
    * https://jira.sonarsource.com/browse/SONAR-7494)
    * @since 2.8
+   * @deprecated in 3.15. Use the property "sonar.web.context" via {@link #setServerProperty(String, String)}.
    */
+  @Deprecated
   public OrchestratorBuilder setContext(String context) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)